content/en/docs/Installation/automated.md

```diff
@@ -128,7 +128,8 @@ total 778M
 $ IMAGE=<scheme://host[:port]/path[:tag]>
 $ mkdir -p files-iso/boot/grub2
 # You can replace this step with your own grub config. This GRUB configuration is the boot menu of the ISO
-$ wget https://raw.githubusercontent.com/kairos-io/kairos/master/overlay/files-iso/boot/grub2/grub.cfg -O files-iso/boot/grub2/grub.cfg
+$ wget https://raw.githubusercontent.com/kairos-io/packages/main/packages/static/grub-config/files/grub.cfg -O files-iso/boot/grub2/grub.cfg
```

Comment:
Should we use the grub from the livecd package?

Reply:
which one?

Reply:
https://raw.githubusercontent.com/kairos-io/packages/main/packages/livecd/grub2/config/grub_live_bios.cfg

Reply:
done